src/export/pdfExporter.ts

```diff
@@ -0,0 +1,81 @@
+import { appendFileSync } from "fs";
+import path = require('path');
+import jsPDF from "jspdf";
+import autoTable from "jspdf-autotable";
+import { PackageInfo } from '../model/packageInfo';
+import { Exporter } from './exporter';
+
+export class PDFExporter implements Exporter {
+    exportBom(packageInfos: PackageInfo[]): void {
+        this.exportToPDF(packageInfos);
+    }
+
+
+    exportToPDF(packageInfos: PackageInfo[]) {
+        var doc = new jsPDF();
+        var col = ["Group", "Name", "Version", "License", "Copyright"];
+        var rows = [];
+
+        var groupPdf: String;
+        var namePdf: String;
+        var versionPdf: String;
+        var licensePdf: String;
+        var copyrightPdf: String;
+
+        for (let i = 0; i < packageInfos.length; i++) {
```

Comment:
Suggestion: you could use packageInfos.foreach(packageInfo => {...}). It would make the code a little more readable.
Also you could think about extracting some blocks like the license for part into there own local function to make it a little more readable.